docs/atomic.md

```diff
@@ -0,0 +1,18 @@
+Atomic host bootstrap
+=====================
+
+Atomic host testing has been done with the network plugin flannel. Change the inventory var `kube_network_plugin: flannel`.
```

Comment:
Atomic by default ships with flannel so for the initial support I would like to start with flannel and maybe expand later.